squid-mixin/README.md

```diff
@@ -0,0 +1,101 @@
+# Squid Mixin
+
+Squid mixin is a set of configurable Grafana dashboards and alerts.
+
+The Squid mixin contains the following dashboards:
+
+- Squid overview
+
+and the following alerts:
+
+- SquidHighPercentageOfHTTPServerRequestErrors
+- SquidHighPercentageOfFTPServerRequestErrors
+- SquidHighPercentageOfOtherServerRequestErrors
+- SquidHighPercentageOfClientRequestErrors
+- SquidLowCacheHitRatio
+
+## Squid Overview
+
+The Squid overview dashboard provides details on both server and client HTTP, FTP, and other requests, caching hits and misses, and both cache and access logs.
+
+![TODO Dashboard screenshots]()
+
+```
```

Comment:
I think this config is missing a text blurb to go along with it. Look at a recent mixin for an example

Reply:
I think this is the way we have been doing it in the absence of the screenshots. 
Heres opensearch:
<img width="1587" alt="image" src="https://github.com/grafana/jsonnet-libs/assets/48131175/c22fc964-c7ff-4de6-aa83-b1890409804f">

Reply:
@armstrmi I'm actually talking about the enableLokiLogs config

Reply:
oh, yup. Will add that bit in.